R/ipr2viz.R

```diff
@@ -212,7 +212,7 @@ plotIPR2Viz <- function(infile_ipr = NULL, infile_full = NULL, accessions = c(),
     analysis_labeler <- analyses %>%
         pivot_wider(names_from = .data$Analysis, values_from = .data$Analysis)
 
-    lookup_tbl_path <- "/data/research/jravilab/common_data/cln_lookup_tbl.tsv"
+    lookup_tbl_path <- "~/awasyn/new_trial/cln_lookup_tbl.tsv"
```

Comment:
I'd suggest changing these references to the home directory, especially when you're referencing your own home directory, with references into the package's filesystem. For example, in this case I'd create a folder called, say, `common_data` under `inst`, put `cln_lookup_tbl.tsv` there, and then change this line to the following:

```suggestion
    lookup_tbl_path <- system.file("common_data", "cln_lookup_tbl.tsv", package = "MolEvolvR", mustWork = TRUE)
```

Reply:
Thanks for this suggestion